pkg/server/plugin/datastore/sql/migration_test.go

```diff
@@ -374,6 +374,13 @@ COMMIT;
 		COMMIT;
 		`,
 		// v11 database entry, in which index was added to federated_registration_entries registered_entry_id field
+		`
```

Comment:
This should be a copy of the database before it receives the migration. The migration test should ensure that the migration test has taken place.

Reply:
Got it, will fix.